packages/pangea-sdk/PangeaCyber.Tests/ITAuditTest.cs

```diff
@@ -908,6 +908,41 @@ public async Task TestLogStream()
         Assert.Equal(nameof(ResponseStatus.Success), response.Status);
     }
 
+    [Fact]
+    public async Task TestExportDownload()
+    {
+        var exportResponse = await generalClient.Export(new ExportRequest
+        {
+            Start = DateTimeOffset.UtcNow.AddDays(-1),
+            End = DateTimeOffset.UtcNow,
+            Verbose = false,
+        });
+        Assert.Equal(nameof(ResponseStatus.Accepted), exportResponse.Status);
+
+        for (int retry = 0; retry < 10; retry++)
+        {
+            try
+            {
+                var response = await generalClient.PollResult<object>(exportResponse.RequestId);
+                if (response.IsOK)
+                {
+                    break;
+                }
+            }
+            catch (PangeaAPIException error) when (error.Response.Status == nameof(ResponseStatus.Accepted) || error.Response.Status == nameof(ResponseStatus.NotFound))
+            {
+                // Allow.
+            }
+
+            await Task.Delay(3 * 1000);
+        }
+
+        var dlResponse = await generalClient.DownloadResults(new DownloadRequest { RequestID = exportResponse.RequestId });
```

Comment:
Here you are assuming you exited `for` on `break` and not because of `retry >= 10`, right? I guess it will work anyway due to this call will fail with 202, but I think would be cleared to check that exit condition.

Reply:
Added an assertion for that.